inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -94,10 +94,9 @@ def set_default_output_files(self):
         """Set default files generated by backend post-processing."""
 
         return [
-            "pressure_field.vtk", "streamlines.vtk", "vertical_slice_0.vtk",
-            "vertical_slice_1.vtk", "vertical_slice_2.vtk",
-            "force_coefficients.csv", "stderr.txt", "stdout.txt",
-            "horizontal_slice.vtk"
+            "pressure_field.vtk", "streamlines.vtk", "stdout.txt", "stderr.txt",
+            "force_coefficients.csv", "xy_slice.vtk", "xz_flow_slice.vtk",
+            "yz_flow_slice.vtk", "constant/triSurface/object.obj"
```

Comment:
This function seems to *get* the default output files and not *set* them. Rename it?

Reply:
Also, it should be a dispatch method, since this will be different for different simulators.

Reply:
I will keep this as a set_default... since it actually sets the default output files you want to download later on. It does not fetch them from anywhere, so get also does not make sense in terms of files.

Reply:
Moreover, I think it can be default for all simulators, since we are the ones deciding the names of the default files and what they look like. The only one here that does not abide to that is the object.obj, but later on can also be simplified.

Reply:
However, I am not computing or deriving something from the object... Should I just set it as an attribute then?

Reply:
I think it should be a method (because you need it to be dispatched differently based on the simulator in use, for instance).

You are *getting* something unique to the object, which I think justifies naming the method as `get_...`.

Reply:
Alrighty then!